roles/ovirt-image-template/README.md

```diff
@@ -28,6 +31,18 @@ Role Variables
 | template_disk_interface | virtio           | Interface of the template disk. |
 | template_timeout   | 600                   | Amount of time to wait for the template to be created. |
 | template_nics      | {name: nic1, profile_name: ovirtmgmt, interface: virtio} | List of dictionaries that specify the NICs of template. |
+| external_templates   | UNDEF               | List of dictionaries that describes the templates |
```

Comment:
I don't like this `external_templates` var. This isn't consistent. This looping should be done on Ansible side: https://github.com/ansible/ansible/issues/24921 someday in future. I think it brings the advatange of asynchronousness for example and simplifies the role.

So if you need that role, can you just create you own role, which won't be part of this repo, so we don't need to support it, and let's add it once users need it? Users still can create what they want in post/pre tasks, those roles just wraps module into role, so it don't do anything basically. Let's talk about it on Monday meeting.